src/io/tradetrust/bill-of-lading/1.0/bill-of-lading.v3.json

```diff
@@ -0,0 +1,67 @@
+{
```

Comment:
Maybe should consider adding it's a context in the file name (instead of v3)

Feel free to ignore this comment if you think the current filename is ok i will approve.

If you update please do it to the other context too

Reply:
updated from .v3 to -context.